test/util.js

```diff
@@ -0,0 +1,9 @@
+'use strict'
```

Comment:
You can use [streaming-iterables collect()](https://github.com/bustle/streaming-iterables#collect)